metrics/github/cli.py

```diff
@@ -69,10 +70,12 @@ def pr_throughput(ctx, org, date):
 
     with TimescaleDBWriter(GitHubPullRequests) as writer:
         opened_prs = api.prs_opened_on_date(org, date)
+        opened_prs = drop_archived_prs(opened_prs)
```

Comment:
I don't see the need for this filter. There won't be any newly-opened PRs for an archived repo, will there?